src/database/tables.js

```diff
@@ -0,0 +1,16 @@
+export const createRecipeTable = (db) => {
+    db.query(`CREATE TABLE IF NOT EXISTS recipes (
```

Comment:
where is .sql file

Reply:
Nowhere, i creating everything in code, like a ORM, i I treat it as a entity :)

Reply:
That was not a task. You need it for later tasks. Orm generates sql migrations based on code for example. It's different approach